src/StaticWebAssetsSdk/Tasks/WriteImportMapToHtml.cs

```diff
@@ -46,6 +46,8 @@ public partial class WriteImportMapToHtml : Task
 
     private static readonly Regex _importMapRegex = new Regex(@"<script\s+type=""importmap""\s*>\s*</script>");
 
+    private static readonly Regex _preloadRegex = new Regex(@"<link\s+rel=""preload""\s*[/]?>");
```

Comment:
I did not mention this at the time for the initial PR because I likely missed it. But can we have a bunch of unit tests to validate these regexes match exactly what we want?